README.md

```diff
@@ -48,3 +48,28 @@ npm start
 After it boots up load the entry at [notifications.localhost:8888](notifications.localhost:8888) in your browser.
 
 [react]: https://facebook.github.io/react/
+
+### Tag, Release and Publish
+
+To create a new version, you can run this command
+
+```bash
+npm version npm version [<newversion> | major | minor | patch] --no-git-tag-version
+```
+
+that will update the `package.json` and the `package-lock.json` files with the new version. The `npm version` command can also create a new commit and a tag for us, but it does not follow our naming conventions.
+Then, you need to commit the changes and create a new git tag. Replace `<newversion>` with the version generated by `npm version`.
+
+```bash
+git commit -a -m "Bump to version <newversion>"
```

Comment:
Probably missing a step here, to review/merge the PR.

Similarly we'd probably want to tag from master vs the branch.